azure-pipelines.yml

```diff
@@ -61,83 +80,88 @@ jobs:
 
 
 #
-# Debug build
+# Debug build (CI)
 #
-
-- template: eng/platform-matrix.yml
-  parameters:
-    jobTemplate: build-job.yml
-    buildConfig: debug
+- ${{ if eq(variables['System.TeamProject'], 'public') }}:
```

Comment:
Is there any case for an internal CI build?

Reply:
I assume no, however I decided to leave it out, as the ci internal build for quickly iterating now while bringing it up is useful.